lib/kredis/types/set.rb

```diff
@@ -18,9 +18,9 @@ def remove(*members)
   end
 
   def replace(*members)
-    multi do
-      del
-      add members
+    multi do |pipeline|
+      pipeline.del key
+      pipeline.sadd key, types_to_strings(members, typed) if members.flatten.any?
```

Comment:
Need to extract `types_to_strings(members, typed) if members.flatten.any?` into a method. It's repeated three times.